data-models/ambiance/illuminance/schema.json

```
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/ambiance/illuminance/schema.json",
+    "$defs": {
+        "light": {
+            "measurementType": "ambiance/illuminance/light/lux",
+            "title": "Light",
+            "unit": "lux",
+            "type": "integer",
+            "description": "The light in lux."
+        },
+        "brightness": {
+            "measurementType": "ambiance/illuminance/brightness/percent",
+            "title": "Brightness",
+            "unit": "%",
+            "type": "integer",
```

```suggestion
            "type": "number",
```

In accordance with the other cases for percentages.